### PR TITLE
Added the getAngle function to TraCI (#10901).

### PR DESCRIPTION
**Pull request by bcoueraud87**

### CODE REVIEW

**Comment by behrisch** [comment]
you should add a test for the edge as well but it may be enough to do this in python (complex/traci)

**Comment by behrisch** [comment]
Is there any reason why you switched from libsumo::INVALID_DOUBLE_VALUE back to -1

**Comment by behrisch** [comment]
The traci/get_variable/Lane test still uses -1. Maybe just test with an explicit value here. Also the doc string in _lane.py is wrong. The result is not the angle from the current position to the end. It is the inclination of the straight line segment at the given position (except for the default case where it is from begin to end).

**Comment by namdre** [comment]
I'd rather say it's the orientation because inclination is more suggestive of the z axis

**Comment by bcoueraud87** [comment]
> The traci/get_variable/Lane test still uses -1. Maybe just test with an explicit value here. Also the doc string in _lane.py is wrong. The result is not the angle from the current position to the end. It is the inclination of the straight line segment at the given position (except for the default case where it is from begin to end).

Thanks I will make the changes and sorry for the mistakes.

**Comment by behrisch** [comment]
Another point: SUMO allows for special district edges without lanes. In this case we should return INVALID_DOUBLE_VALUE

**Comment by bcoueraud87** [comment]
> Another point: SUMO allows for special district edges without lanes. In this case we should return INVALID_DOUBLE_VALUE

I will do that as well then.

File: src/libsumo/Edge.cpp [hunk 1]
```diff
@@ -281,6 +282,14 @@ Edge::getPendingVehicles(const std::string& edgeID) {
     return vehIDs;
 }
 
+
+double
+Edge::getAngle(const std::string& edgeID, double relativePosition) {
+    MSLane* lane = getEdge(edgeID)->getLanes()[0];
+    return lane ? Lane::getAngle(lane->getID()) : libsumo::INVALID_DOUBLE_VALUE;
```
Comment (behrisch) [inline src/libsumo/Edge.cpp:289]: This is not correct. If the edge has no lanes, the vector is empty
